features/323-login-id/api.md

```diff
@@ -0,0 +1,182 @@
+# API Summary
+
+## SDK API
```

Comment:
I think we need an API to set primary login ID?

Primary ID now is based on the array inside loginIDkey, but we have no way to manipulate that.

Can we add a short section here about what is primary email/sms ID too? So what it is for, and how to set the primary login ID.

Reply:
As mentioned in stage 3 spec, 'primary login ID' is a concept only existing during the signup process; it is not persisted. It is just a term (like a variable name) used to refer to the welcome email destinations.

Reply:
Hm okay, so it is only for welcome email.